Payments/Overview/02_Features.md

```diff
@@ -0,0 +1,44 @@
+
+## Automated import
+With the *Payments* module of Actindo, pre-authorized payments are captured automatically at the moment orders are delivered. If products are announced, but not yet launched, customers can reserve a product and pay later when the shipment has started.
+In case of returns or credit notes, refund requests are automatically sent out to the original means of payment. By using the 4-eyes principle, you can optionally double-check the refunds, for example, with specific user permissions depending on the invoice amount.  
+The single steps of the payments process are usually integrated in an order workflow. The workflow controls that no products are shipped without a payment commitment from the customer.
+
+
+## Connecting with payment service providers
+Youu can flexibly connect various payment service providers such as Adyen&trade;, Paypal&trade; to the *Actindo Core1 Platform*. Depending on the payment service provider you decide for, credit cards, bank transactions, and other standard payment methods are supported implicitly.
+
+
+## Importing bank statements and automatic posting
+Bank statements and reports from payment service providers can be automatically processed and posted to the *Payment Processing* module and therefore by passing the *Payment Processing* module to the Actindo *Accounting* module or a third-party Accounting. In case of prepayments, you can hold the shipment of orders until the invoice amount is received. It is then and automatically released for shipment after payment has been made.
+
+
+## Payment processing and accounting linked 
+ The Actindo *Payment* module connects the business accounts at your various payment service providers and the Actindo *Payment Processing* module. In being the smart connection between those two modules, the Actindo *Payment* module helps you to process orders faster and more efficiently. Thanks to automated payment checks, incoming payments are registered directly, and orders could therefore be processed immediately. Refunds can be made just as quickly and automatically. By linking payment and accounting, you do not only minimize the workload for your employees, but also offer your customers a smooth and fast service.
+
+ **Payments interaction with other modules** 
+
+![Payments interaction with other modules](../../Assets/Screenshots/Payments/Overview/PaymentsProcessModules.png "[Payments interaction with other modules]")
+
+
+## Managing all payments centrally 
+The payment options for customers are diverse, and so are the payment service providers that exist on the market today. In order to be able to offer your customers the widest possible range of payment methods - from invoice to credit card to direct debit - cooperation with various providers is required and provided by Actindo. This enables the management of all payments and refunds centrally via the *Actindo Core1 Platform*.  
```

Comment:
- from invoice to credit card to direct debit - --> Das sollte eigentlich zwischen em dash (oder Kommas, Klammern) geschrieben werden: https://www.merriam-webster.com/words-at-play/em-dash-en-dash-how-to-use

Reply:
Done